packages/recommend-vdom/src/TrendingFacets.tsx

```diff
@@ -13,9 +13,7 @@ export function createTrendingFacetsComponent({
   createElement,
   Fragment,
 }: Renderer) {
-  return function TrendingFacets<TObject>(
-    props: TrendingComponentProps<TObject>
-  ) {
+  return function TrendingFacets(props: TrendingComponentProps<{}>) {
```

Comment:
This would be the equivalent to before, as the other generic is optional.

```suggestion
  return function TrendingFacets(props: TrendingComponentProps) {
```